doc/Requirements Specification/Parts/Models.lyx

```diff
@@ -179,7 +179,7 @@ status open
 
 
 \backslash
-glslabel{software architecture}{architecture}
+gls{software architecture}
```

Comment:
You mean `\glslink{software architecture}{architecture}`, don't you? I changed it to this.